docs/docs/api/Tasks/fetchSingleTask.md

```diff
@@ -20,11 +20,13 @@ This is a **protected route**, a **valid JWT is required** in the header field
 #### Header
 
 ```
-Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
+Cookie:
+token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
 ```
+
 #### Parameters
 
-  id: task_id 
+id: task_id
 
 ### Possible Responses
```

Comment:
For fetchSingleTask, I would also include an example of what the data looks like rather than just an empty JSON with a comment of `// Task information`

Reply:
Fixed: Changes are reflected in recent commits